src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Colors [] colors = Colors.values();
+
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        return String.valueOf(colors[random.nextInt(colors.length - 1)]);
```

Comment:
Please, use enum's name() method instead, as described [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants). Also, should you really decrement colors.length by 1?